cmd/kubectl-k8ssandra/register/register_test.go

```diff
@@ -0,0 +1,178 @@
+package register
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	configapi "github.com/k8ssandra/k8ssandra-operator/apis/config/v1beta1"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestRegister(t *testing.T) {
+	require.New(t)
+	client1 := (*multiEnv)[0].GetClient("source-namespace")
+	client2 := (*multiEnv)[1].GetClient("dest-namespace")
+
+	if err := client1.Create((*multiEnv)[0].Context, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "source-namespace"}}); err != nil {
+		t.Fatal(err)
```

Comment:
issue: If we have require, use that and not t.Fatal(). Test error handling should be consistent.

Reply:
This is another case where it makes little difference. This error is indeed fatal, nothing can proceed after that so there's no point making it a soft requirement. 

Regardless, if it makes you feel the code is more consistent, I will implement this feedback.

Reply:
This is not about fatal / not fatal. require is fatal, the test will stop at that point. It's not assert, it's require. Either don't use require at all unlike all our other programs or use it everywhere you would otherwise use t.Fail / t.Fatal / t.FailNow